mllm/backends/qnn/passes/CustomOpPatterns.hpp

```
@@ -0,0 +1,22 @@
+
+#include "mllm/backends/qnn/op/QNNBaseOp.hpp"
+#include "mllm/compile/ir/linalg/Op.hpp"
+#include "mllm/compile/ir/Node.hpp"
+#include "mllm/compile/ir/tensor/Value.hpp"
+#include <vector>
+
+namespace mllm::qnn {
+
+class QNNDequantizeAddPattern : public QNNBasePattern {
+ public:
+  bool isMatch(const ir::op_ptr_t& op) override { return op->isa_<ir::linalg::CustomizedOp>(); }
```

Clang (14.0.6)

[error] parameter name 'op' is too short, expected at least 3 characters

(readability-identifier-length,-warnings-as-errors)